service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/util/DateUtilTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.nhs.adaptors.oneoneone.cda.report.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.util.Date;
+
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class DateUtilTest {
+
+    @Test
+    public void shouldParseForCorrectDateFormat() {
+        String dateAsString =  "201201021234+00";
```

Comment:
There's a double space between = and ". Please pay attention to formatting. We will have less work to do once checkstyle is on.

Reply:
sorry, I have the style check on:)